src/Reflection/ReflectionClass.php

```diff
@@ -542,6 +535,45 @@ function (ReflectionClassConstant $constant) : string {
     }
 
     /**
+     * Get an associative array of the defined constants in this class,
+     * with keys as constant names and values as {@see ReflectionClassConstant} objects.
+     *
+     * @return ReflectionClassConstant[]
+     */
+    public function getReflectionConstants() : array
+    {
+        $allReflectionConstants = array_merge(
+            array_values($this->getImmediateReflectionConstants()),
+            ...array_map(
```

Comment:
Can't remember if `array_merge()` accepts a single argument. If not, then this code will crash with no ancestors nor interfaces. The simplest solution is to add an empty array as second parameter